### PR TITLE
Split AccountSettingVC into 5 screens

### PR DESCRIPTION
**Pull request by or-else**
AccountSetting VC was refactored into five screens to support adding more options (specifically Blocked accounts need to be added to keep iOS on par with web and Android).

A couple of bugs were fixed: incorrect handling of future and making UI changes from background thread.

### CODE REVIEW

File: Tinodios/SettingsGeneralViewController.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//
+//  SettingsGeneralViewController.swift
+//  Tinodios
+//
+//  Copyright © 2020 Tinode. All rights reserved.
+//
+
+import UIKit
+
+class SettingsPersonalViewController: UITableViewController {
```
Comment (aforge) [inline Tinodios/SettingsGeneralViewController.swift:10]: Why do you need to declare it here?
It's also declared in Tinodios/SettingsPersonalViewController.swift.

Comment (or-else) [reply Tinodios/SettingsGeneralViewController.swift]: Fixed. It was some kind of fluke in Xcode - the file was not showing in the UI.

File: Tinodios/SettingsPersonalViewController.swift [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+            actionTarget: self)
+        UiUtils.setupTapRecognizer(
+            forView: manageContacts,
+            action: #selector(SettingsPersonalViewController.addContactClicked),
+            actionTarget: self)
+
+        self.imagePicker = ImagePicker(presentationController: self, delegate: self, editable: true)
+    }
+
+    private func reloadData() {
```
Comment (aforge) [inline Tinodios/SettingsPersonalViewController.swift:61]: Can you also refresh the contacts table please?

File: Tinodios/SettingsSecurityViewController.swift [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    }
+
+    private func logout() {
+        Cache.log.info("SettingsSecurityVC - logging out")
+        UiUtils.logoutAndRouteToLoginVC()
+    }
+
+    private func deleteAccount() {
+        Cache.log.info("SettingsSecurityVC - deleting account")
+        tinode.delCurrentUser(hard: true).thenApply { _ in
```
Comment (aforge) [inline Tinodios/SettingsSecurityViewController.swift:202]: Can you add a failure handler please?

File: Tinodios/SettingsPersonalViewController.swift (hunk 2 above)
Comment (or-else) [reply Tinodios/SettingsPersonalViewController.swift:61]: There are some problems with contacts which I want to address in the next PR. For instance, contacts don't show up at all without a connection.

File: Tinodios/SettingsSecurityViewController.swift (hunk 3 above)
Comment (or-else) [reply Tinodios/SettingsSecurityViewController.swift]: done

**Pull request merged**